samples/matter/light_bulb/Kconfig.sysbuild

```diff
@@ -5,9 +5,10 @@
 #
 
 source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+source "${ZEPHYR_NRF_MODULE_DIR}/boards/shields/nrf7002ek/Kconfig.shield"
 
 config NRF_DEFAULT_MULTIPROTOCOL
-	default y if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp")
+	default y if ($(BOARD) = "nrf5340dk_nrf5340_cpuapp") && !SHIELD_NRF7002EK
 
 config NRF_DEFAULT_BLUETOOTH
-	default y if ($(BOARD) = "nrf7002dk_nrf5340_cpuapp")
+	default y if ($(BOARD) = "nrf7002dk_nrf5340_cpuapp") || SHIELD_NRF7002EK
```

Comment:
I do not find it acceptable for sysbuild to be loading board specific shield files

Reply:
Discussed f2f. Will be reverted.